metrics.go

```diff
@@ -0,0 +1,91 @@
+package es
+
+import (
+	"sync"
+	"time"
+
+	"github.com/elastic/go-elasticsearch/v7/esapi"
+
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+var clientDurationSummary *prometheus.SummaryVec
+
+func init() {
+	clientDurationSummary = prometheus.NewSummaryVec(
+		prometheus.SummaryOpts{
+			Name: "search_operations_durations_seconds",
+			Help: "search_operations_durations_seconds",
+		},
+		[]string{"client_name", "method"},
```

Comment:
lets add result ok/fail and error code if possible?

Reply:
added status ok/error and response code